### PR TITLE
fix: use correct color names

### PR DESCRIPTION
**Pull request by omBratteng**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://as-696-fix-colours.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/PlusUser.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const PlusUser = ({
 }: Props): ReactElement => {
   return (
     <Typography
-      className={classNames('flex text-accent-bacon-default', className)}
+      className={classNames('text-action-plus-default flex', className)}
```
Comment (rebelchris) [inline packages/shared/src/components/PlusUser.tsx:19]: We actually have `TypographyColor.Plus`

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/shared/src/components/PlusUser.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/components/PlusUser.tsx:19]: I don't see it 🙈 
Do you have it in your branch?
https://github.com/dailydotdev/apps/blob/b1714c4f80cc68cb224daa0d8d6ca4665df4d62d/packages/shared/src/components/typography/Typography.tsx#L40-L48

Comment (omBratteng) [reply packages/shared/src/components/PlusUser.tsx:19]: Yeah, found it there
https://github.com/dailydotdev/apps/blob/3eba23b2b80028cec4809f2805f8f1f5c7878e42/packages/shared/src/components/typography/Typography.tsx#L48

Comment (rebelchris) [reply packages/shared/src/components/PlusUser.tsx:19]: Oh right, my bad thought I pushed it on the other one already.
Maybe base it on my one so long?

Comment (rebelchris) [reply packages/shared/src/components/PlusUser.tsx:19]: Or take that change in here already :)

Comment (omBratteng) [reply packages/shared/src/components/PlusUser.tsx:19]: I'll rebase it over yours, and change merge target to your branch

**Review by rebelchris — APPROVED** [review]

**Pull request merged**